src/lib.rs

```diff
@@ -69,15 +69,96 @@
     unused_parens,
     while_true
 )]
-use ansi_term::{
-    Color::{Cyan, Purple, Red},
-    Style,
-};
 use std::env;
 use std::fmt;
 use std::fs::File;
 use std::io::{BufRead, BufReader, ErrorKind};
 use tracing_error::SpanTrace;
+use once_cell::sync::OnceCell;
+use owo_colors::{style, Style, XtermColors};
+
+static STYLES: OnceCell<Styles> = OnceCell::new();
+
+// XXX now that we support text effects, "color scheme" doesn't seem accurate anymore. Let me know if you'd prefer a different name. The good thing about `Styles` is, that it's pretty short.
+
+/// A struct that represents styles that should be used by `color_spantrace`
+#[derive(Debug, Copy, Clone, Default)]
+pub struct Styles {
+    file: Style,
+    line_number: Style,
+    target: Style,
+    fields: Style,
+    active_line: Style,
+}
+
+impl Styles {
+    /// Create blank styles
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    /// Styles for a dark background. This is the default
+    pub fn dark() -> Self {
+        Self {
+            file:         style().purple(),
+            line_number:  style().purple(),
+            active_line:  style().white().bold(),
+
+            // XXX your trait from this file (which I have removed now) produced "xterm" colors. `owo_colors` documents the `xterm` module as follows: "XTerm 256-bit colors. Not as widely supported as standard ANSI but contains 240 more colors." (see *). Maybe it would be best to switch to something else now. In this case, the colors also have to be changed in `owo_colors`
```

Comment:
Yea, lets definitely switch to ANSI. I didn't realize I was using something other than ansi tbh, I think the intention was to have use the same variants of red and blue being used in color-eyre for error msgs and suggestions, respectively.

Reply:
Finally, I have some time to work on this again! :)

Okay, I changed the xterm colors to "bright_*". As tests anyway fail, until the new version of `owo_colors` is released, I'll keep the old test data. If you run the test, you see the new colors are almost identical to the old ones.

Let me know if I just should remove the old test data.